api/v1alpha1/awsadapterconfig_types.go

```diff
@@ -82,26 +82,26 @@ type EKSCompute struct {
 
 // EKSNodeGroup contains info of the EKS cluster's node group
 type EKSNodeGroup struct {
-	Name               string                          `json:"name,omitempty"`
-	NodegroupArn       *string                         `json:"nodeGroupArn,omitempty"`
-	NodeRole           *string                         `json:"nodeRole,omitempty"`
-	CreatedAt          string                          `json:"createdAt,omitempty"`
-	Status             string                          `json:"status,omitempty"`
-	DiskSize           *int32                          `json:"diskSize,omitempty"`
-	AMIType            string                          `json:"amiType,omitempty"`
-	CapacityType       string                          `json:"capacityType,omitempty"`
-	AMIReleaseVersion  *string                         `json:"amiReleaseVersion,omitempty"`
-	Subnets            []string                        `json:"subnets,omitempty"`
-	InstanceTypes      []string                        `json:"instanceTypes,omitempty"`
-	UpdateConfig       *EKSNodeGroupUpdateConfig       `json:"updateConfig,omitempty"`
-	ScalingConfig      *EKSNodeGroupScalingConfig      `json:"scalingConfig,omitempty"`
-	LaunchTemplate     *EC2LaunchTemplate              `json:"launchTemplate,omitempty"`
-	RemoteAccessConfig *EKSNodeGroupRemoteAccessConfig `json:"remoteAccessConfig,omitempty"`
-	Resources          *EKSNodeGroupResources          `json:"resources,omitempty"`
-	HealthIssues       []*EKSNodeGroupHealthIssue      `json:"healthIssues,omitempty"`
-	Taints             []*EKSNodeGroupTaint            `json:"taints,omitempty"`
-	Labels             map[string]string               `json:"labels,omitempty"`
-	Tags               map[string]string               `json:"tags,omitempty"`
+	Name                string                          `json:"name,omitempty"`
+	NodegroupArn        *string                         `json:"nodeGroupArn,omitempty"`
+	NodeRole            *string                         `json:"nodeRole,omitempty"`
+	CreatedAt           string                          `json:"createdAt,omitempty"`
+	Status              string                          `json:"status,omitempty"`
+	DiskSize            *int32                          `json:"diskSize,omitempty"`
+	AMIType             string                          `json:"amiType,omitempty"`
+	CapacityType        string                          `json:"capacityType,omitempty"`
+	AMIReleaseVersion   *string                         `json:"amiReleaseVersion,omitempty"`
```

Comment:
I think we should call this NodeGroupReleaseVersion or just ReleaseVersion (@anusha94 ?)